### PR TITLE
RHMAP-14876 - Missing steps to configure WFM Demo into RHMAP

### PR DESCRIPTION
**Pull request by camilamacedo86**
JIRA : https://issues.jboss.org/browse/RHMAP-14876

### CODE REVIEW

**Comment by camilamacedo86** [comment]
@nialldonnellyfh could you check and merge it for me please ?

**Comment by camilamacedo86** [comment]
@finp Could you check and merge it?  Missed step identified into a case. 
JIRA: https://issues.jboss.org/browse/RHMAP-14876

File: getting-started.adoc [hunk 1]
```diff
@@ -87,6 +87,24 @@ RHMAP provides a Raincatcher project template to help you run the demonstration.
 
 .. Click *Deploy Cloud App*.
 
+. Upgrade the database into *WFM Demo Project* and *WFM Auth Service*:
```
Comment (pwright) [inline getting-started.adoc]: suggestion:
Upgrade the database for the *Cloud App* and the *WFM Auth Service*:

File: getting-started.adoc [hunk 2]
```diff
@@ -87,6 +87,24 @@ RHMAP provides a Raincatcher project template to help you run the demonstration.
...
+
+.. Select the *Projects* header menu item.
+
+.. Go to the *Cloud Application*
```
Comment (pwright) [inline getting-started.adoc]: Click on the Cloud App

File: getting-started.adoc [hunk 3]
```diff
@@ -87,6 +87,24 @@ RHMAP provides a Raincatcher project template to help you run the demonstration.
...
+
+.. Select Data Browser from the left menu
+
+.. Click on *Upgrade Database*
```
Comment (pwright) [inline getting-started.adoc]: Click the *Upgrade Database* button in the top right corner, and confirm by clicking *Upgrade Now*. Wait until the upgrade process finishes.

**Review by pwright — COMMENTED** [review]
just trying to keep the doc consistent. I'm thinking 'Navigate to the *Projects* area' to be consistent with the later text. if you want, I can merge this now and tidy up later?

File: getting-started.adoc (hunk 1 above)
Comment (camilamacedo86) [reply getting-started.adoc]: Done.

File: getting-started.adoc (hunk 2 above)
Comment (camilamacedo86) [reply getting-started.adoc]: Done.

**Comment by camilamacedo86** [comment]
@finp changes made. I don't have the access to merge in this repo. Do you have? Could you do it for me?

File: getting-started.adoc [hunk 4]
```diff
@@ -87,6 +87,24 @@ RHMAP provides a Raincatcher project template to help you run the demonstration.
...
+.. Click on the *Upgrade Database* button in the top right corner, and confirm by clicking Upgrade Now. Wait until the upgrade process finishes
+
+.. Navigate to the *Services & APIs* area
+
+.. Select the *WFM Auth Service* created
+
+.. Select Data Browser from the left menu
+
+.. Click on the *Upgrade Database* button in the top right corner, and confirm by clicking Upgrade Now. Wait until the upgrade process finishes
+
```
Comment (wtrocki) [inline getting-started.adoc:107]: Missing deploy step. After migration service needs to be deployed again.

**Comment by wtrocki** [comment]
@camilamacedo86 Thanks for contribution. Changes reviewed and verified. 

@finp feel free to mention some possible issues if any. We can fix them in separate jira.

**Review by wtrocki — APPROVED** [review]

**Pull request merged**